packages/client/src/exampleTypes/pivot.js

```diff
@@ -69,6 +69,8 @@ let maybeRemoveSelectedRows = (extend, node) => {
 let resetExpansions = (extend, node) => {
   extend(node, {
     expansions: [],
+    hasResults: false,
+    context: { results: {} },
```

Comment:
Does reset expansions actually clear all results? 🤔

Reply:
It actually should. Otherwise, we can see all results in the newly configured report. And those old results are showing when we load new results and can have wrong columns formatting etc because new pivot configuration applied while having results for the previous configuration